check-log/check-log.go

```diff
@@ -113,9 +114,14 @@ func run(args []string) *checkers.Checker {
 
 	warnNum := int64(0)
 	critNum := int64(0)
+	missingNum := int64(0)
```

Comment:
How about declaring `var missingFiles []string` and check `len(missingFiles)` instead of `missingNum` for more friendly error messages.